01-builder-visualization/exercises/twitter-dogs.md

```diff
@@ -0,0 +1,199 @@
+
+* Click on the `POP-UP` tab
+* In the `CLICK` pop-up menu, set style to `IMAGE`
+* Check off the following items, and drag them into the order they are listed:
+  * `actor_image`
+  * `actor_displayname`
+  * `actor_preferredusername`
+  * `body`
+  * `link`
+
+![pop-up click settings](imgs/01-dog-03.png)
+
+* In the `HOVER` pop-up menu, set style to `LIGHT`
+* Check the box for `actor_displayname` and change the type field next to it to `Who is Tweeting?`
+
+![pop-up click settings](imgs/01-dog-04.png)
+
+* Go back to the `CLICK` pop-up menu, and use the bottom toggle to switch to HTML mode
+
+##### Deleting the heading "actor_preferredusername" and adding an @ before the user name
+
+* Find this piece of code:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order2">
+    <h5 class="CDB-infowindow-subtitle">actor_preferredusername</h5>
+    <h4 class="CDB-infowindow-title">{{actor_preferredusername}}</h4>
+```
+and alter it as follows:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order2">
+    <h5 class="CDB-infowindow-subtitle"></h5>
+    <h4 class="CDB-infowindow-title">@{{actor_preferredusername}}</h4>
+```
+
+##### Deleting the heading "body"
+
+* Find this piece of code:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order3">
+<h5 class="CDB-infowindow-subtitle">body</h5>
+<h4 class="CDB-infowindow-title">{{body}}</h4>
+```
+and alter it as follows:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order3">
+<h5 class="CDB-infowindow-subtitle"></h5>
+<h4 class="CDB-infowindow-title">{{body}}</h4>
+```
+
+##### Making a clickable hyperlink that will open the linked tweet in a new browser tab
+
+* Find this piece of code:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order4">
+    <h5 class="CDB-infowindow-subtitle">link</h5>
+    <h4 class="CDB-infowindow-title">{{link}}</h4>
+```
+and alter it as follows:
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order4">
+    <h5 class="CDB-infowindow-subtitle"></h5>
+    <h4 class="CDB-infowindow-title"><a href="{{link}}" target="_blank">View Tweet</a></h4>
+```
+
+* Click `APPLY`.
+
+![pop-up click HTML](imgs/01-dog-05.png)
+
+<hr>
+
+## 3. Widgets <a name="widgets"></a>
+
+### 3. 1. Add widgets
+
+* Navigate to  the **WIDGETS** pane.
+  * You will notice that your time slider is already present.
+* Click `ADD`.
+  * In the options of the **CATEGORY** tab, check the *category_terms* widget. Make sure you are adding a widget from the `A0` layer node.
+  * In the options of the **FORMULA** tab, check `retweetcount` and `actor_followerscount`, again making sure you are adding the widget with the source as `A0`.
+* Click on `CONTINUE`.
+
+### 3. 2. Order and edit widgets
+
+* Navigate back to the **WIDGETS** pane.
+* **Click the *time widget*.**
+  - Change the title to **When are people tweeting?**.
+  - You can change the color of the widget bars now too if you'd like.
```

Comment:
Be more specific here - something like - You can also change the color of the widget timeseries bars by selecting the FILL color option at the bottom of the widget pane.